src/navigation/components/NavigationItem/components/NavigationDropdown.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+import {NavigationItemProps} from '../NavigationItem';
+import {Content} from './Content';
+import {ToggleArrow} from '../../../../components';
+
+import {block} from '../../../../utils';
+import {getMediaImage} from '../../../../components/Media/Image/utils';
+
+import {DropdownItemData} from '../../../../models';
+
+import '../NavigationItem.scss';
+
+const b = block('navigation-item');
```

Comment:
If we want to have one class nn different components, we should add a component-wrapper with children, for example